query/query.ts

```diff
@@ -49,6 +49,7 @@ export interface QueryConfig {
   encoder?: (arg: unknown) => EncodedArg;
   name?: string;
   text: string;
+  snakeToCamel?: boolean;
```

Comment:
Add the option to the QueryObjectConfig interface, since QueryConfig applies to queryArray which contains no query names